lib/ability.rb

```diff
@@ -126,8 +128,10 @@ def initialize(user, resource, controller = nil)
         end
 
         can [:administer], User
+
         if controller.is_a?(UsersController) || controller.is_a?(SearchController)
           can :manage, User
+          cannot([:edit, :update], User)
```

Comment:
If we do pursue adding strong params, I'd like to consider changing this and the following permissions (the ones that completely prevent edit/update rather than preventing editing internal users) to use a whitelist instead of `manage`.